packages/suite/src/components/suite/WelcomeLayout.tsx

```diff
@@ -93,9 +96,20 @@ const StyledTrezorLink = styled(TrezorLink)`
 `;
 
 const SettingsWrapper = styled.div`
+    position: absolute;
     align-self: flex-end;
 `;
 
+const ChildrenWrapper = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    align-items: center;
```

Comment:
Is this deliberate? I would not change it.

Before:
![Screenshot 2023-11-13 at 12 59 38](https://github.com/trezor/trezor-suite/assets/42465546/b2b4dc8c-0863-45fc-afc9-25eca56db421)
After:
![Screenshot 2023-11-13 at 13 13 04](https://github.com/trezor/trezor-suite/assets/42465546/be4d95e9-1985-46c2-ac98-9a6e1d79967e)

Reply:
Not deliberate but IMO looks so much nicer in the `WelcomeLayout` when it's the same width as the other elements, like prerequisites, and not suddenly super wide.

Reply:
I am not yet convinced because it can also cause the width to change between these two cards which is not so nice:


https://github.com/trezor/trezor-suite/assets/42465546/bb9704a8-c6e2-41f2-b86d-5b80b8a0258e

Reply:
36271bb70e05b6d9d60307da7f3927eedfd28a71